models/staging/products/products.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+
+sources:
+  - name: coffee_shop
+    project: analytics-engineers-club
+
+    tables:
+      - name: products
+        columns:
+          - name: id
+            tests:
+              - unique
+              - not_null
+          - name: name
+            tests:
+              - not_null
+          - name: category
+            tests:
```

Comment:
It depends on the frequency of change but one other test you could do is the accepted_values test on product category which is available by default because we only have 3 categories. I guess coffee shops could add more product categories but probably wouldn't change often.